CHANGES/6673.feature

```diff
@@ -0,0 +1 @@
+Enabled dispatching customization for importing collections
```

Comment:
I think this should be a `misc` entry because user's capabilities are not changed by it.